tests/utils/test_Http.py

```diff
@@ -109,17 +110,23 @@ def test_PUT_json(self):
 
         assert response['form'] == {'aaa': '42', 'bbb': '123'}
 
-    def test_port_is_open__port_is_not_open(self):
+    def test_is_port_open_port_is_open__port_is_not_open(self):
```

Comment:
add an double _ to split the names of the methods tested

if not how should I read the `test_is_port_open_port_is_open` of `test_is_port_open_port_is_open__port_is_not_open`